hack/spc_build.sh

```diff
@@ -37,24 +36,10 @@ make GOPATH=$(pwd)/_build:$(pwd) all TAGS="seccomp containers_image_ostree_stub"
 GOPATH=$(pwd)/_build:$(pwd) go test -c -tags "seccomp `./btrfs_tag.sh` `./btrfs_installed_tag.sh` `./libdm_tag.sh` `./ostree_tag.sh` `./selinux_tag.sh`" ./cmd/buildah
 tmp=$(mktemp -d); mkdir $tmp/root $tmp/runroot; PATH="$PATH" ./buildah.test -test.v -root $tmp/root -runroot $tmp/runroot -storage-driver vfs -signature-policy $(pwd)/tests/policy.json -registries-conf $(pwd)/tests/registries.conf
 
-echo "docker login to local registry..."
+echo "Logging in to local registry..."
 echo testpassword | docker login localhost:5000 --username testuser --password-stdin
 
-echo "docker build dockerfile..."
-docker build -f hack/Dockerfile -t docker-test-image .
-
-echo "buildah bud dockerfile..."
-./buildah --registries-conf tests/registries.conf bud --file hack/Dockerfile -t buildah-test-image .
-
-echo "buildah tag buildah-test-image..."
-./buildah tag buildah-test-image localhost:5000/buildah-test-image
-
-echo "buildah push buildah-test-image..."
-./buildah push --cert-dir /home/travis/auth --tls-verify=false --authfile /root/.docker/config.json buildah-test-image localhost:5000/buildah-test-image
-
-echo "docker pull buildah-test-image..."
-docker pull localhost:5000/buildah-test-image
-
-echo "Running container-diff..."
-container-diff diff --type=rpm daemon://localhost:5000/buildah-test-image daemon://docker-test-image
+echo "Running conformance tests..."
+GOPATH=$(pwd) go test -v ./tests/conformance
```

Comment:
Are the removed *test-image tests taken care of elsewhere?

Reply:
What I was commenting on here was on the left side of the github review screen, a number of tests were removed ala:

```
echo "buildah tag buildah-test-image..."
./buildah tag buildah-test-image localhost:5000/buildah-test-image
```
I'm not sure that they are covered by the conformance tests.  If you think we're hip, then I'm hip.